src/platform/packages/private/kbn-grid-layout/grid/grid_row/grid_row.tsx

```diff
@@ -197,3 +192,15 @@ export const GridRow = ({
     </div>
   );
 };
+
+const styles = {
+  gridRow: css({
+    display: 'grid',
+    position: 'relative',
+    justifyItems: 'stretch',
+    transition: 'background-color 300ms linear',
+  }),
+  fullHeight: css({
+    height: '100%',
+  }),
+};
```

Comment:
If there is no reliance on EUI theme, then it is more performant to store styles as minimizable objects outside of the React component so that it is not parsed on every render. See https://engineering.deptagency.com/optimizing-emotion-in-react#optimization-i-avoid-doing-construction-of-styles-in-render-functions for a more thorough explanation here